src/DatePicker/Calendar.js

```diff
@@ -25,6 +26,53 @@ import {
 
 const daysArray = [...Array(7)];
 
+const styleSheet = createStyleSheet('Calendar', (theme) => {
```

Comment:
Not good for performance, imperative methods, parent-child communication and again performance :grin: 

We'll probably soon use react-mixout, maybe then, we'll use it to inject class names too. but not for now